implementations/micrometer-registry-humio/src/main/java/io/micrometer/humio/HumioConfig.java

```diff
@@ -45,14 +45,6 @@ default String uri() {
         return v == null ? "https://cloud.humio.com" : v;
     }
 
-    /**
-     * @return The repository name to write metrics to.
-     */
-    default String repository() {
```

Comment:
Rather than suddenly deleting this, which may break compatibility for users, let's mark it as deprecated and note that it is no longer used and will be removed.

Reply:
We _can_ put it back in as a deprecated method, but it will be completely ignore in the code. Your call.

Reply:
Yes, that would be preferable. If you make this change but revert the change in the other classes, they result in compiler errors because they claim to `@Override` a method not declared in `HumioConfig` anymore. This is what users would experience, but it is not a great upgrade experience. We can deprecate it in 1.3 (and stop using it for anything) and remove it in 1.4 so users will have been able to notice the deprecation and remove its usage for a smooth upgrade to 1.4.